Add possibility to define a custom `rem()` function (#520)
JosephusPaye/Keen-UI: Customization.md

Some components also have component-specific variables that can be found in their source files. Those can be overridden as well.

> You can also override the `rem` function by defining a `rem-custom` function which would be used instead for calculating the size of components components. Please refer to the original function in [`keen-ui/src/styles/utils.scss`](.../src/styles/utils.scss).

Reviewer: Some suggestions:

```suggestion
You can also override the `rem` function by defining a `rem-custom` function to be used instead for calculating the size of components, as described below. Please refer to the original function in [`src/styles/utils.scss`](./src/styles/utils.scss).
```
